patient-connect/200_ImplementingSSR.md

```diff
@@ -0,0 +1,158 @@
+---
+name: Implementing Server-Side Rendering (SSR) for Provider and Location Profile Pages (WIP 🚧)
+---
+
+# Implementing Server-Side Rendering (SSR) for Provider and Location Profile Pages (WIP 🚧)
+
+## Introduction
+**The Server-Side Rendering (SSR)** feature in Care Search is an optional enhancement that allows provider and location information to be included in the initial HTML response from your server. This improves the chances that search engines will index your profile pages correctly, as key content is included at the time of page load.
```

Comment:
The Server-Side Rendering (SSR)** feature in Search Client is an optional enhancement that allows provider and location content to be included in the initial HTML response from the server. Since the server is fully rendering the page content before sending it to the browser, its generally easier for search engines to crawl and index it once it does since it will already be present in the page's source code at load time.